<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hello_Qt_OpenCV</class>
 <widget class="QMainWindow" name="Hello_Qt_OpenCV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello_Qt_OpenCV</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widgetMain" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widgetInput" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lbInput">
            <property name="text">
             <string>Input Image:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leInput"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnInput">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Filter type</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="rbtnMedianBlur">
            <property name="text">
             <string>Median Blur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbtnGaussianBlur">
            <property name="text">
             <string>Gaussian Blur</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetOutput" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lbOutput">
            <property name="text">
             <string>Output Image:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leOutput"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnOutput">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbDisplayAfterSave">
         <property name="text">
          <string>Display image after saving</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
